src/Widgets/ImageWidget/DimensionsEditor.scss

```diff
@@ -84,12 +84,84 @@
     min-width: 40px;
     text-align: center;
 
-    &[disabled] {
+    &:disabled {
       appearance: none;
       cursor: unset;
     }
   }
 
+  .enum_attribute {
+    margin: 0;
+    padding: 0 0 3px 0;
+    display: grid;
+    gap: 6px;
+    grid-template-columns: repeat(auto-fit, minmax(105px, 1fr));
+
+    button {
+      position: relative;
+      border-style: none;
+      border-radius: 3px;
+      border-left: 30px solid #f0f0f0;
+      box-shadow: 0 0 4px 1px rgba(0, 0, 0, 0.1);
+      padding: 4px 8px;
+      line-height: 15px;
+      display: flex;
+      flex-flow: column;
+      background: #fafafa;
+      color: #555;
+      font-size: 13px;
+      font-weight: normal;
+    }
+
+    button .attribute-preview {
+      width: 45px;
+      height: 30px;
+      margin: 2px auto;
+      &.contain {
+        background: no-repeat center/ contain url('object-fit-contain.svg');
+      }
+      &.cover {
+        background: no-repeat center/ contain url('object-fit-cover.svg');
+      }
+    }
+
+    button span {
+      color: #555;
+      font-size: 11px;
+      text-align: center;
+      display: block;
+      margin: auto;
+    }
+
+    button:not(:disabled):hover {
+      background: #eee;
+      cursor: pointer;
+    }
+
+    button.enum_attribute_active {
+      border-left: 30px solid var(--jr-blue) !important;
+      box-shadow: 0 0 0 2px var(--jr-blue) inset;
```

Comment:
Where is `var(--jr-blue)` defined? I can only see `var(--jr-blue-light)` in `scrivitoExtensions.scss`.

Answering my own question: In `scrivito_editing.css`.

I'm not sure, if we want to rely on css-variables from the SDK.